unlock-protocol-com/blog/alphatweet-announce.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Unlock Labs Launches the ALPHAtweet Experiment
+authorName: Christopher Carfi
+publishDate: May 9, 2023
+description: ALPHAtweet is an experiment in web3 business models where creators and their community both share in the upside of creating and sharing content across social networks.
+image: /images/blog/alphatweet/alphatweet-share.png
+---
+
+![alphatweet-share.png](/images/blog/alphatweet/alphatweet-share.png)
+
+_**ALPHA by Unlock Labs** is an initiative to explore new business models and opportunities enabled by blockchain technologies and using Unlock Protocol. It is an iterative series of experiments with the intended goals of learning and sharing information and experiences as part of the open-source Unlock Protocol project._
+
+## This experiment is called ALPHAtweet
+
+ALPHAtweet enables you and your followers make money through your content and sharing. It’s powered by Unlock Protocol. Going a bit deeper into this experiment, an “Alpha” is a piece of paid content that is being shared, where **both the creator and the sharer participate in the benefits of sharing when someone unlocks the Alpha**.
```

Comment:
I am not sure but I think this sentence misses a "to"?

"ALPHAtweet enables you and your followers **to** make money through your content and sharing."